workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -25,6 +25,15 @@ internal static class DownloadCoreSdk
 
         private const string DownloadForceMenuItem = "SpatialOS/Download CoreSdk (force)";
 
+        private static readonly List<PluginDirectoryCompatibility> PluginsCompatibilityList = new List<PluginDirectoryCompatibility>
+        {
+            PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/OSX", new List<BuildTarget> { BuildTarget.StandaloneOSX }, true),
```

Comment:
A named parameter will make the meaning of all the `true/false` arguments clearer.

Reply:
Good point, I am still getting used to C# conventions.
Would it be a better idea to only name the last parameter, assuming that path and platform list are implied from the method call, or is it better to explicitly state all parameters?

Reply:
Yeah, sorry, that's what I meant. Just the last parameter needs to be named; I think the context is obvious for the other parameters.